docker-storage-setup.sh

```diff
@@ -720,6 +719,21 @@ unit: sectors
 
 ${dev}1 : start=     2048, size=  ${size}, Id=8e
 EOF
+}
+
+create_partition_parted(){
+  local dev="$1"
+  parted $dev --script mklabel dos mkpart primary 0% 100%
```

Comment:
I have changed `mklabel` from `dos` to `msdos` based on this documentation:
https://www.gnu.org/software/parted/manual/html_node/mklabel.html